composites/basic/Tabs.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
```

Comment:
This component should not exist. I believe there should be a Tab component, and the SubNavigation component should map the Tab component. This way a superfluous div is rendered.